### PR TITLE
Tutorial: Package a dependency from ADO

### PR DESCRIPTION
**Pull request by vicroms**

### CODE REVIEW

File: vcpkg/produce/package-ado-repository.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/copyright" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+```
+
+Creating a port to package an Azure DevOps follows the same process as packaging
+any Git-hosted package.
+
+The `portfile.cmake` provided here is but a template to follow for your own
+packages, you may require additional commands to build your package and copy the
+output into vcpkg's installation folder.
+
```
[the remark this answers is not in the document]
Comment (vicroms) [reply vcpkg/produce/package-ado-repository.md:92]: Yeah, this tutorial needs a concrete example. I'll ask Robert about setting up a public ADO repository for the sample library you created to use it in this tutorial.

File: vcpkg/produce/package-ado-repository.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+> [!div class="checklist"]
+> * [Create a port directory](#1---create-a-port-directory)
+> * [Create the port's manifest](#2---create-the-ports-manifest)
+> * [Create the port's recipe script](#3---create-the-ports-recipe-script)
+
+## Prerequisites
+
+* A terminal
+* A C++ copmpiler
```
Comment (AugP) [inline vcpkg/produce/package-ado-repository.md:32]:
```suggestion
* A C++ compiler
```

File: vcpkg/produce/package-ado-repository.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+* vcpkg
+* Git
+* A library hosted in an Azure DevOps repository
+
+## 1 - Create a port directory
+
+Create a new directory with the name you want to give to your port.
```
Comment (AugP) [inline vcpkg/produce/package-ado-repository.md:39]: Where should this directory be created? On your machine? In ADO? Any restrictions for the location of the new directory? E.g. a place you have read/write access to.

File: vcpkg/produce/package-ado-repository.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+## 2 - Create the port's manifest
+
+Inside the port's directory create a file named `vcpkg.json`, replace the
+placeholders with the appropiate values for your package.
```
Comment (AugP) [inline vcpkg/produce/package-ado-repository.md:44]: Maybe the reader should be instructed to run "vcpkg new" and guided about what changes to make to the file after? I think we want users to establish a habit of using vcpkg commands wherever possible to do things automatically. While it makes little difference in a tutorial where they can copy and paste, it's a useful habit to have if they want to do it again at a later time.

File: vcpkg/produce/package-ado-repository.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+}
+```
+
+See the [`vcpkg.json` reference](../reference/vcpkg-json.md) for detailed
+descriptions of each property in the manifest file.
+
+## 3 - Create the port's recipe script
+
+Inside the port's directory create a file named `portfile.cmake`, replace the
+placeholders with the appropiate values for your package.
```
Comment (AugP) [inline vcpkg/produce/package-ado-repository.md:64]:
```suggestion
Inside the port's directory, create a file named `portfile.cmake` and replace the
placeholders with the appropiate values for your package.
```

File: vcpkg/produce/package-ado-repository.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+file(INSTALL ${HEADER_FILES} DESTINATION "${CURRENT_PACKAGES_DIR}/include/${PORT}")
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/copyright" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+```
+
+Creating a port to package an Azure DevOps follows the same process as packaging
+any Git-hosted package.
+
+The `portfile.cmake` provided here is but a template to follow for your own
+packages, you may require additional commands to build your package and copy the
+output into vcpkg's installation folder.
```
Comment (AugP) [inline vcpkg/produce/package-ado-repository.md:91]:
```suggestion
The `portfile.cmake` provided here is a template to follow for your own packages. You may require additional commands to build your package and copy the output into your vcpkg installation folder.
```

**Pull request closed without merging**